docs/contributions/documentation_guidelines.md

```diff
@@ -23,61 +26,75 @@ This document explains how to contribute to the LibreChat documentation by writi
 - You can use HTML to add additional features to a document.
 
 ## Document Metadata
+
 - Add metadata in the header of your document following this format:
+
 ```yaml
 ---
 title: 😊 Document Title
 description: This description will be used in social cards and search engine results.
 weight: 0
 ---
 ```
+
 - `title:` Begin with an emoji representing your new document, followed by a descriptive title.
 - `description:` A brief description of the document's content.
 - `weight:` Setting the weight in the document metadata will influence its position in the table of contents. Lowest weights are placed first. If not set, it defaults to `0`. When multiple docs have the same weight, they are sorted alphabetically.
 
 ## Important Notes
+
 - **⚠️Keep the documentation organized and structured⚠️**
 - Do not add unrelated information to an existing document. Create a new one if needed.
 - Upload all assets (images, files) directly from GitHub's web interface when editing the document.
 
 > **Tip:** Go to the LibreChat repository, find a conversation, and paste an image from your clipboard into the text input box. It will automatically be converted into a URL you can use in your document. (Then exit the page without actually posting the comment.😉)
 
-  > Get the link from a text input box:
-  ![image](https://github.com/danny-avila/LibreChat/assets/32828263/c1612f93-a6c0-4af7-9965-9f83872cff00)
-  Or upload directly from the web UI:
-  ![image](https://github.com/danny-avila/LibreChat/assets/32828263/4f138ab4-31a5-4fae-a459-5335e5ff25a8)
+> Get the link from a text input box:
+> ![image](https://github.com/danny-avila/LibreChat/assets/32828263/c1612f93-a6c0-4af7-9965-9f83872cff00)
+> Or upload directly from the web UI:
+> ![image](https://github.com/danny-avila/LibreChat/assets/32828263/4f138ab4-31a5-4fae-a459-5335e5ff25a8)
 
 ## Testing New Documents
+
 - When adding new documents, it is important to test them locally using MkDocs to ensure correct formatting and proper organization in the table of contents (index.md and the left panel of the category).
 
-### Setup MkDocs Locally 
-- Requirement: Python 3.x
+### Setup MkDocs Locally
+
+- Requirement: Python 3.3 and later (on older version you will need to install virtualenv)
 
 #### Material for MkDocs Installation
+
 - We are using MkDocs Material and multiple plugins. All of them are required to properly test new documentation.
+
```

Comment:
We should probably specify the location of the `requirements.txt` file here

Reply:
maybe specify the requirements.txt path in the command? 
e.g. `pip install -r ./docs/src/requirements.txt`